documentation/src/docs/basic-reader-and-writer.md

```diff
@@ -52,178 +48,167 @@ public void run(String routingKey, String message) {
-we can then go on and create the Stream in lines 5-8. 
-
-The StreamManager needs 3 things to create a Stream, the Scope's name, the
-Stream's name and a StreamConfiguration.  The most interesting task is to create
-the StreamConfiguration.
-
-Like many objects in Pravega, a Stream takes a configuration object that allows
+| (static) `create` | (URI controller)                                                  | Given a URI to one of the Pravega Controller instances in the Pravega Cluster, create a Stream Manager object. |
+| `createScope`     | (String scopeName)                                                | Creates a Scope with the given name.                                                                           |
+|                 |                                                                   | Returns **_True_** if the Scope is created, returns **_False_** if the Scope already exists.                                                                                                               |
+|                 |                                                                   | This method can be called even if the Stream is already existing.                                                                                                              |
+| `deleteScope`     | (String scopeName)                                                | Deletes a Scope with the given name.                                                                           |
+|                 |                                                                   | Returns **_True_** if the scope was deleted, returns **_False_** otherwise.                                                                                                               |
+|                 |                                                                   | **Note:** If the Scope contains Streams, the `deleteScope` operation will fail with an exception.                                                                                                               |
+|                 |                                                                   | If we delete a nonexistent Scope, the method will succeed and return **_False_**.                                                                                                               |
+| `createStream`    | (String scopeName, String streamName, StreamConfiguration config) | Create a Stream within a given Scope.                                                                          |
+|                 |                                                                   | Both Scope name and Stream name are limited using the following characters: ( Letters (a-z A-Z), numbers (0-9) and delimiters like (. , -) are allowed). punctuation).                                                                                                               |
```

Comment:
`( Letters (a-z A-Z), numbers (0-9) and delimiters like (. , -) are allowed). punctuation).`:
- Why this is in a parenthesis? 
- `delimiters like  (. , -)`: we have to be precise here, so I would enumerate only the symbols that are allowed. That is: `and delimiters: "." and "-".` The rest of symbols are not allowed to build Stream names.

Reply:
Accepted suggestion